project.clj

```diff
@@ -53,5 +53,4 @@
                                    [puppetlabs/trapperkeeper :classifier "test" :scope "test"]]}
              :testutils {:source-paths ^:replace ["test/unit" "test/integration"]}}
 
-  :plugins [[lein-parent "0.3.1"]]
-  )
+  :plugins [[lein-parent "0.3.1" :exclusions [org.clojure/clojure]]])
```

Comment:
Ah, I forgot I had this in here. I pulled down the master branch and tried to run the tests, and got pedantic errors before I made any changes. I'm assuming it's something funky with my lein parent plugin or something, just hadn't figured the problem out yet